doc/man3/EVP_SKEY.pod

```diff
@@ -65,9 +65,10 @@ that gets passed the value of I<export_cbarg>.  See L<openssl-core.h(7)> for
 more information about the callback. Note that the L<OSSL_PARAM(3)> array that
 is passed to the callback is not persistent after the callback returns.
 
-The EVP_SKEY_get_raw_key() function copies raw key bytes to the passed buffer
-and sets the key len. The operation can fail when the underlying key
-management doesn't support export of the secret key.
+The EVP_SKEY_get_raw_key() returns a pointer to a raw key bytes to the passed
```

Comment:
`... to an array of raw key bytes ...`

However seeing that, I would say this function should be renamed to EVP_SKEY_get0_raw_key

Reply:
Done